### DISCUSSION TITLE
Retention error for repositories containing '/'.

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [goharbor/harbor#20398] [Q&A] [answered]
Hi,
There is a retention error for repositories containing '/'.
{"errors":[{"code":"NOT_FOUND","message":"repository hy/test%2Fbusybox not found"}]}

No issues when using `dry run: true`.

How can I fix this?

**Comment by wy65701436** [comment, marked as answer]
Please file an issue for it and attach your screenshot of the retention policy.
